core/src/parameter.jl

```diff
@@ -74,13 +74,18 @@ end
 Base.to_index(id::NodeID) = Int(id.value)
 
 const ScalarInterpolation = LinearInterpolation{
-    ReadOnlyVector{Float64, Vector{Float64}},
-    ReadOnlyVector{Float64, Vector{Float64}},
+    Vector{Float64},
+    Vector{Float64},
     Vector{Float64},
     Vector{Float64},
     Float64,
 }
 
+set_zero!(v) = fill!(v, zero(eltype(v)))
+const LBC = LazyBufferCache{Returns{Int}, typeof(set_zero!)}
+FixedSizeLazyBufferCache(len::Int)::LBC =
```

Comment:
Capitalizing this confused me a bit thinking it is a type. How about just calling this function `cache`? Could also use a docstring, to explain why you need the initializer.

`Return` is neat, hadn't used it before.